tests/integration/modules/test_key.py

```diff
@@ -14,6 +14,7 @@ def test_key_finger(self):
         test key.finger to ensure we receive a valid fingerprint
         '''
         out = self.run_function('key.finger')
+        print(out)
```

Comment:
I think you left a debugging line in here. Can you remove this?

Reply:
Done.